README.md

```diff
@@ -41,6 +41,15 @@ Download the appropriate [release](https://github.com/ethanmoffat/EndlessClient/
 
 After installing, clone (or fork+clone) this repository locally and open the solution in your IDE of choice for your platform.
 
+### Building on Mac
+
+1. Download and install the [.NET 6.0 SDK (x64)](https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/sdk-6.0.403-macos-x64-installer).
```

Comment:
Is the .net core 3.1 runtime also required?

Reply:
I don't believe so. Upgrading the monogame content builder task makes it work with 6

Reply:
Oh that makes sense. I think I'll make a follow-up PR to update XNAControls to 3.8.1 and fix the Linux execution since that's a really nice change to have